lib/pure/typetraits.nim

```diff
@@ -30,6 +30,14 @@ runnableExamples:
   type C[T] = enum h0 = 2, h1 = 4
   assert C[float] is HoleyEnum
 
+runnableExamples:
+  # `toSigned`, `toUnsigned`
+  assert int8.toSigned is int8
+  assert int8.toUnsigned is uint8
+  assert uint.toUnsigned is uint
+  assert int.toUnsigned is uint
+  assert $int32.toUnsigned == "uint32"
```

Comment:
```suggestion
  assert int32.toUnsigned is uint32
```

Reply:
ok, i ended up removing that line and adding a comment instead; the reason i had added it in the 1st place is that depending on how you define toUnsigned,toSigned, you could get a different string representation, so I wanted to show this one doesn't suffer from this; but I think it's a reasonable assumption to make from this API that it works correctly with `$`, and the tests exercise this anyways.